source/Server.Contracts/Endpoints/DeploymentActionContainer.cs

```diff
@@ -8,11 +11,14 @@ public class DeploymentActionContainer
     {
         public const string DefaultTag = "latest";
         public string? Image { get; set; }
-        public string? FeedId { get; set; }
+
+        [JsonProperty("FeedId")] // This is named FeedId for backward-compatibility as we don't yet want to change the underlying database JSON/schema.
+        [OclName("feed")]
+        public FeedIdOrName? FeedIdOrName { get; set; }
 
         public bool IsConfigured()
         {
-            return !string.IsNullOrEmpty(Image) && !string.IsNullOrEmpty(FeedId);
+            return !string.IsNullOrEmpty(Image) && !string.IsNullOrEmpty(FeedIdOrName?.Value);
```

Comment:
Taking suggestions for a better way to check if these TinyTypes are set.

Reply:
I think those other places were a mistake we made while learning about these changes. TinyType can never contain a blank string, its constructor will throw if you try to do that, so the check should always just need to be `FeedIdOrName is not null`. Note: `is null` and `is not null` are also the preferred way to do this, vs `==` and `!=`.

Reply:
> Note: `is null` and `is not null` are also the preferred way to do this, vs `==` and `!=`

This makes me happy.